mlflow/cli.py

```diff
@@ -599,6 +597,14 @@ def gc(older_than, backend_store_uri, run_ids):
 except ImportError as e:
     pass
 
+try:
+    import mlflow.pipelines.cli  # pylint: disable=unused-import
+
+    cli.add_command(mlflow.pipelines.cli.commands)
```

Comment:
Add a TODO to remove the silent import failure once pipelines is brought into the core release.

Reply:
I think pipeline is already part of the core releases right since it is part of the main init of mlflow which should already added mlflow as part of the core package?
Let me know if I am missing something.

Reply:
In that case why do we need this try-catch clause?

Reply:
This is what @dbczumar recommended as a clean way to avoid the numpy import errors that we got in skinny API. This is similar to what we do for other cases to solve this problem.

Reply:
Got it. SGTM!